sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+WITH events_unnested AS (
```

Comment:
Could we add a column e.g. `is_desktop_interaction` that returns [this condition](https://github.com/mozilla/private-bigquery-etl/blob/49023493d12e4ff380a07a78ecbb2b7e648f2c85/sql/moz-fx-data-shared-prod/ads_derived/interaction_native_aggregates_client_telemetry_hourly_v1/query.sql#L116-L119). It'd be used to create a desktop revenue aggregate in the model.